src/components/Dropdown/MultiSelectDropdown.jsx

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Downshift from 'downshift';
+import { DropdownInput, DropdownMenu } from '@ndla/forms';
+import { itemToString } from '../../util/downShiftHelpers';
+
+class MultiSelectDropdown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: [],
+      isOpen: false,
+    };
+    this.addNewTag = this.addNewTag.bind(this);
+    this.onInputChange = this.onInputChange.bind(this);
+    this.removeItem = this.removeItem.bind(this);
+    this.onValueChange = this.onValueChange.bind(this);
+    this.handleStateChange = this.handleStateChange.bind(this);
+  }
+
+  onValueChange(newValue) {
+    const { onChange, value, textField } = this.props;
+    onChange({
+      target: {
+        name: 'tags',
+        value: [...value, itemToString(newValue, textField)],
+      },
+    });
+    this.setState({ inputValue: '', isOpen: false });
+  }
+
+  addNewTag(e) {
+    e.preventDefault();
+    const { value } = this.props;
+    const { inputValue } = this.state;
+    if (value.includes(inputValue)) {
+      return;
+    }
+    this.onValueChange({ title: inputValue });
+  }
+
+  onInputChange(e) {
```

Comment:
Bare meg som liker evt og err isteden for e?

Reply:
Ja :)